test/Database/LSMTree/Class/Normal.hs

```diff
@@ -271,7 +270,7 @@ withCursor offset hdl = bracket (newCursor offset hdl) (closeCursor (Proxy @h))
 instance IsSession R.Session where
     data SessionArgs R.Session m where
       SessionArgs ::
-           forall m h. (MonadFix m, MonadMask m, MonadST m, Typeable h)
+           forall m h. (MonadMask m, MonadST m, Typeable h)
         => HasFS m h -> HasBlockIO m h -> FsPath
         -> SessionArgs R.Session m
```

Comment:
These constraints on `m` are not needed (including `MonadMask` and `MonadST`), because they are already included in `IOLike`. Could you remove them?